client/asset/btc/spv.go

```diff
@@ -1823,13 +1825,17 @@ func (w *spvWallet) getTransaction(txHash *chainhash.Hash) (*GetTransactionResul
 	// This is what the JSON-RPC does (and has since at least May 2018).
 	details, err := w.wallet.walletTransaction(txHash)
 	if err != nil {
+		if errors.Is(err, WalletTransactionNotFound) {
+			return nil, asset.CoinNotFoundError // for the asset.Wallet interface
+		}
```

Comment:
Yeah.  I had this as a separate commit since it was mostly unrelated: https://github.com/decred/dcrdex/pull/1607/commits/cb6fe1908b02e3e0ff65bcc9de10063b333a2678